packages/modular-scripts/src/buildPackage/makeBundle.ts

```diff
@@ -220,11 +225,11 @@ export async function makeBundle(
   ].filter((dep) => !localFileNames.has(dep));
 
   if (missingDependenciesWithoutLocalFileNames.length > 0) {
-    throw new Error(
-      `Missing dependencies: ${missingDependenciesWithoutLocalFileNames.join(
-        ', ',
-      )};`,
-    );
+    logger.error(`Missing dependencies found:`);
+    missingDependenciesWithoutLocalFileNames.forEach((missingImport) => {
+      logger.error(`  ${missingImport}`);
+    });
+    throw new Error(`Missing dependencies found.`);
```

Comment:
`Missing dependencies found` would be printed twice here.

Reply:
yeah good spot - I'll remove the first comment.